zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/AQLQueryParser.scala

```diff
@@ -197,9 +201,12 @@ class AQLQueryParser extends IQueryParser {
   private def parseSelect(select: SelectStatement,
                           varMap: Map[String, AQLVar],
                           isInGroup: Boolean,
-                          sourceVar: String = "$g"
-                         ): (String, String) = {
+                          sourceVar: String = "$g",
+                          selectVar: String
```

Comment:
for the `sourceVar` and `selectVar`, I feel we could add the `innerSourceVar`, `outerSelectVar` to be specific.